packages/pluggableWidgets/bar-chart-native/src/BarChart.editorConfig.ts

```diff
@@ -0,0 +1,96 @@
+import { hideNestedPropertiesIn, Problem, Properties } from "@widgets-resources/piw-utils";
+
+import { BarChartPreviewProps } from "../typings/BarChartProps";
+
+export function getProperties(widget: BarChartPreviewProps, defaultProperties: Properties): Properties {
+    widget.barSeries.forEach((series, index) => {
```

Comment:
In all other widgets we use values as default variable for getProperties functions..

Reply:
I changed it because `widget` made more sense to me, but I'll revert to `values` to keep it consistent 👍

Reply:
👍